.gitignore

```diff
@@ -11,6 +11,9 @@ pids
 *.seed
 *.pid.lock
 
+# macOS directory files
+.DS_Store
```

Comment:
There are a _lot_ more of these kinds of files, lol.  Fine for now, but I prefer for people to keep these things in their own local `~/.gitignore` files, since those are applied globally.

It would be fine to just go ahead and add:
```
._*
/.Trash/
/.Spotlight-V100/
```
to this, just to have a complete index.

Reply:
… When have you ever managed to get a .Trash folder in a git repo? Out of curiosity.

And I normally have .DS_Store in my global gitignore, but I seem to have broken said file on my machine currently.